src/ripple/app/misc/impl/LoadFeeTrack.cpp

```diff
@@ -80,11 +82,30 @@ LoadFeeTrack::lowerLocalFee ()
 
 //------------------------------------------------------------------------------
 
-// Scale from fee units to millionths of a ripple
-std::uint64_t
-scaleFeeBase(std::uint64_t fee, Fees const& fees)
+template <class T1, class T2,
+    class = std::enable_if_t <
+        std::is_integral<T1>::value &&
+        std::is_unsigned<T1>::value &&
+        sizeof(T1) <= sizeof(std::uint64_t) >,
```

Comment:
Given that we're dealing with `unsigned` integers, this shouldn't matter, _however_ your suggestion better expresses the intent of the current code. That, alone, would be enough to get me to change.

However, consider that this code will be rewritten once we get C++17 to the following blob, should we just punt?

```
template <class T1, class T2,
    class = std::enable_if_t <
        std::is_integral_v<T1> &&
        std::is_integral_v<T2>>
>
void lowestTerms(T1& a,  T2& b)
{
    if (auto const gcd = std::gcd(a, b))
    {
        a /= gcd;
        b /= gcd;
    }
}
```